docs/en/latest/plugins/chaitin-waf.md

```diff
@@ -0,0 +1,253 @@
+|--------------------------|---------------|----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| match                    | array[object] | false    |               | The list of matching rules, default is empty                                                                                                                                                                                                                                                                                                              |
+| match.vars               | array[array]  | false    |               | List of variables to match for filtering requests for conditional traffic split. It is in the format `{variable operator value}`. For example, `{"arg_name", "==", "json"}`. The variables here are consistent with NGINX internal variables. For details on supported operators, [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list). |
+| add_header               | bool          | false    | true          | Whether to add response headers                                                                                                                                                                                                                                                                                                                           |
+| add_debug_header         | bool          | false    | false         | Whether or not to add debugging headers, effective when `add_header` is `true`.                                                                                                                                                                                                                                                                           |
+| config                   | object        | false    |               | Configuration of the Chaitin SafeLine WAF service. When the route is not configured, the parameters configured in the metadata are used.                                                                                                                                                                                                                  |
+| config.connect_timeout   | integer       | false    |               | connect timeout, in milliseconds, integer                                                                                                                                                                                                                                                                                                                 |
```

Comment:
```suggestion
| config.connect_timeout   | integer       | false    |               | connect timeout, in milliseconds                                                                                                                                                                                                                                                                                                                 |
```

Same for below. **Remove all `integer` from the description** as the data types have already been specified in `Type`.